File: app/src/main/java/com/worldventures/dreamtrips/wallet/analytics/wizard/PowerOnAction.java
package com.worldventures.dreamtrips.wallet.analytics.wizard;

import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.dreamtrips.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:Power on your flye SmartCard",
                navigationState = true,
                trackers = AdobeTracker.TRACKER_KEY)
public class PowerOnAction extends WalletAnalyticsAction {
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/wizard/PowerOnAction.kt
package com.worldventures.wallet.analytics.wizard

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:Power on your flye SmartCard", navigationState = true, trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class PowerOnAction : WalletAnalyticsAction()
